### PR TITLE
#164798195  Configuring Postgres Database Engine

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?
Configure Postgres database
#### Description of Task to be completed?
It configures the Author's Haven Django application to use Postgres DBEngine  instead of the default SQLite database
#### How should this be manually tested?
- Fetch this branch `git fetch origin ch-apply-postregsdb-164798195`
- Checkout to the branch `git checkout ch-apply-postregsdb-164798195`
- create a `.env` file at the root of the project
- copy the content of the `.env_config_sample `file  on the root of the project and paste it in the created .env file and configure the database as per to your wish
- Run `python manage.py migrate ` to create database models to the set up database
- Run the application `python manage.py runserver` to start server
#### What are the relevant pivotal tracker stories?
[#164798195](https://www.pivotaltracker.com/story/show/164798195)

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ jwt = "*"
 django-cors-headers = "*"
 djangorestframework = "*"
 django-extensions = "*"
+psycopg2 = "*"
```
Comment (ja-odur) [inline Pipfile]: @felixkiryowa  use `psycopg2-binary`

File: authors/settings.py [hunk 2]
```diff
@@ -81,8 +81,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```
Comment (ja-odur) [inline authors/settings.py]: @felixkiryowa Please the `dj_database_uri` library to handle this

**Review by ja-odur — CHANGES_REQUESTED** [review]

**Comment by ja-odur** [comment]
@felixkiryowa Please add the steps to fetch and checkout to this branch in your `How to manually test` section.

**Comment by dannylwe** [comment]
> #### What does this PR do?
> Configure Postgres database
> 
> #### Description of Task to be completed?
> It configures the Author's Haven Django application to use Postgres DBEngine instead of the default SQLite database
> 
> #### How should this be manually tested?
> * create a `.env` file at the root of the project
> * source the .env file using this command in a terminal of the project
>   `source .env`
> * copy the content of the `env_file` on the root of the project and paste it in the created .env file and configure the database as per to your wish
> * Run `python manage.py makemigrations`
> * Run `python manage.py migrate ` to create database models to the set up database
> 
> #### What are the relevant pivotal tracker stories?
> [#164798195] (https://www.pivotaltracker.com/story/show/164798195)

@felixkiryowa Kindly properly link the PT story

**Pull request merged**